sycl/test-e2e/DeprecatedFeatures/queue_old_interop.cpp

```diff
@@ -36,7 +36,7 @@ int main() {
     queue q;
     print_queue_info(q);
 
-  } catch (device_error e) {
+  } catch (const exception &e) {
```

Comment:
Should we also check error code here now?

Reply:
I don't think it matters here - the test goes through the no exception code path here. I think it's some debug printing again.